src/components/button/__tests__/Button.test.tsx

```diff
@@ -30,11 +31,7 @@ describe('components/button/Button', () => {
     test('simulates click events', () => {
         const onClickHandler = sinon.spy();
 
-        const wrapper = shallow(<Button onClick={onClickHandler} />);
-
-        const contains = sinon.stub();
-        contains.withArgs('is-disabled').returns(false);
-        wrapper.instance().btnElement = { classList: { contains } };
+        const wrapper = shallow<Button>(<Button onClick={onClickHandler} />);
```

Comment:
Is there a lot of value in adding types in our test files? Or is it purely opt-in? I feel like this will slow productivity in these types of files personally.

Reply:
I think there is value in it in terms of autocomplete, not creating invalid test definitions, etc. but many of our tests will fail because of incomplete mock data. You can see this already if you add `// @flow` to a file.